TheThingsNetwork/lorawan-stack
# Unset LoRaCloud DAS integration completely

#### Summary


Closes https://github.com/TheThingsIndustries/lorawan-stack/issues/2548
References https://github.com/TheThingsIndustries/lorawan-stack/pull/2504
References https://github.com/TheThingsNetwork/lorawan-stack/issues/2630

#### Changes


- Port the delete package SDK functionality
- Use the said functionality when unsetting the LoRaCloud integration
- Change the name of the LoRaCloud DAS form (required for Geolocation, as LoRaCloud is the general service name for both).


#### Testing



_Keep in mind that testing this probably requires a `jssdk:cleanDeps js:cleanDeps`_

- Set the token via the Console and check the network requests that request/response are correct.
- Delete the token via the Console and check that the network request/response are correct

##### Regressions



This does affect the LoRaCloud DAS form, but testing should cover the changes.

#### Checklist
